CommunicationLayer/MessagingFramework/BatteryMessage.cpp

```diff
@@ -0,0 +1,218 @@
+#include "BatteryMessage.h"
+#include "MessageDecodingHelpers.h"
+#include "MessageDefines.h"
+
+using namespace MessageDecodingHelpers;
+
+namespace
+{
+    const int BMU_ALIVE = 1;
```

Comment:
Just going to mark some of these with 'done' so I can keep track of which ones I have completed locally before I push (causing a bunch of these comments to become hidden)
done